pallets/msa/src/lib.rs

```diff
@@ -251,6 +251,29 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::weight((T::WeightInfo::remove_msa_delegation_by_provider(20_000), DispatchClass::Normal, Pays::No))]
```

Comment:
The weight function takes a u32 and MessageSenderId is a u64 so I just guessed at a reasonable number.